packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -4834,11 +4834,11 @@ export declare type SchoolCreateFormValidationValues = {
     name?: ValidationFunction<string>;
     Students?: ValidationFunction<Student>;
 };
-export declare type FormProps<T> = Partial<T> & React.DOMAttributes<HTMLDivElement>;
+export declare type PrimitiveOverrideProps<T> = Partial<T> & React.DOMAttributes<HTMLDivElement>;
```

Comment:
A type override pr is merged in before one of mine and the snapshot get overridden. Update it back to the correct one